tests/scripts/analyze_outcomes.py

```diff
@@ -460,6 +460,7 @@ class DriverVSReference_ecc_no_bignum(outcome_analysis.DriverVSReference):
         ],
         'test_suite_asn1parse': [
             'INTEGER too large for mpi',
+            re.compile(r'Parse integer new,.*'),
```

Comment:
I really don't like having “new” in the test case. Next we know, we'll have the “new new” test cases, and the “really new up-to-date ignore all the other ones” test cases, and so on... Can we call them something meaningful like “raw”?

Reply:
You're right that is laziness on my part, I had meant to come up with a better name. I like 'raw', that sounds fine.